src/cmd-buildextend-metal

```diff
@@ -143,6 +143,17 @@ if [ "${image_type}" = dasd ] || [ "${image_type}" = metal4k ]; then
     ignition_platform_id=metal
 fi
 
+# Firstboot kernel arguments.
+firstboot_kargs="$(python3 -c '
+import sys, yaml;
+LEGACY_DEFAULT = [ "rd.neednet=1", "ip=dhcp,dhcp6" ];
+args = yaml.safe_load(sys.stdin).get("firstboot-kargs", LEGACY_DEFAULT);
+if not args:
```

Comment:
One subtlety here that I missed at first (which contributed to why I originally wanted something more explicit than this) is that it *appears* like this can never be false since we provide a default. Maybe a comment here like: `# firstboot-kargs provided, but is empty` ?

Reply:
(I blame PEP8 for convincing a generation of programmers that an array and a boolean are isomorphic)